### DISCUSSION TITLE
Search results returned in 'PnP - Search Results' webpart is not coming up as expected/relevant to the searched keyword.

### DISCUSSION THREAD

**Discussion by ABritam19** [microsoft-search/pnp-modern-search#3586] [Q&A]
Need help regarding how search result can be ordered by relevance, when searching with a keyword. For example if we search for keyword 'Guidance documents', it is not sorting the results as per the keyword in PnP modern search webpart. We need to surface search results based on the term provided separate from the latest created search results. Also, need to know what can be accomplished with PnP modern search results webparts in sharepoint online.

Note : Created a microsoft ticket, but they mentioned that PnP search webpart is not out of box and not in their scope of help.

**Comment by wobba** [comment]
Kudos to support as they are correct. These web parts do not fall under any support agreement. They are using public available API's with the limitations they have. By default the PnP web part will sort on relevance, and relevance is defined downstream in the API's and nothing we can change.

*Relevance* is a black box with many parameteres we have no control over. Keyword matching in e.g. title and body are just one part of it.
